provisioning_templates/snippet/ansible_provisioning_callback

```diff
@@ -0,0 +1,28 @@
+<%#
+kind: snippet
+name: ansible_provisioning_callback
+-%>
+<% if host_param_true?('ansible_tower_provisioning') -%>
+<%
+  rhel_compatible = @host.operatingsystem.family == 'Redhat' && @host.operatingsystem.name != 'Fedora'
+  os_major = @host.operatingsystem.major.to_i
+  realm_compatible = (@host.operatingsystem.name == 'Fedora' && os_major >= 20) || (rhel_compatible && os_major >= 7)
+-%>
+<% if realm_compatible -%>
+<%= save_to_file('/etc/systemd/system/ansible-callback.service',
+                 snippet('ansible_tower_callback_service')) %>
+# Runs during first boot, removes itself
+/usr/bin/systemctl enable ansible-callback
+<% else -%>
+# Assume systemd is not available
+<% call = <<~CALL
```

Comment:
Given this is duplicated in the finish section, should this be extracted to a snippet as well?

Reply:
It's not 100% duplicated, this prints a command to a script, the one in the finish template is just the `curl` call directly

Reply:
But you can use ```save_to_file``` with the snippet there.